DigitalLearningSolutions.Web/Views/LearningPortal/Shared/_Dates.cshtml

```diff
@@ -64,15 +63,16 @@
       <a class="nhsuk-button nhsuk-button--secondary small-edit-button"
          aria-describedby="@Model.Id-name"
          aria-label="Edit complete by date"
+         asp-action="SetCurrentActionPlanResourceCompleteByDate"

```

Comment:
Can we not do some similar action logic to the SetCompleteByDate view to reduce this if else group?

Reply:
I tried just now but here there's another difference in this case - the name of the route id that is passed. Is it possible to pass in the name of an attribute as a variable?

Reply:
Probably not worth it then, didn't notice the difference before